### PR TITLE
Kan 1 aravind filter services

### PR DESCRIPTION
**Pull request by aravind-digitech**
Added Header.

Added Search feature for services.

Included typing animation in search services component.

Added Dropdown for Search Services.

### CODE REVIEW

File: Frontend/src/main.jsx [hunk 1]
```diff
@@ -2,9 +2,12 @@ import React from 'react'
...
 import './index.css'
+import SiteHeader from './components/header/header.jsx'
 
 ReactDOM.createRoot(document.getElementById('root')).render(
   <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+    {/* <App /> */}
+    {/* <ServiceDropdown></ServiceDropdown> */}
+    <SiteHeader></SiteHeader>
```
Comment (GREEDnim) [inline Frontend/src/main.jsx]: We should have App component here right?

Comment (Murali03M) [inline Frontend/src/main.jsx]: We can revert the changes in the main.jsx file

**Review by GREEDnim — APPROVED** [review]
All good!

File: Frontend/src/main.jsx (hunk 1 above)
Comment (aravind-digitech) [reply Frontend/src/main.jsx]: Yes, I need to revert it

**Review by Murali03M — APPROVED** [review]
Merged with development branch

**Review by Murali03M — APPROVED** [review]
